tools/github_helper/main.go

```diff
@@ -21,6 +21,8 @@ var (
 	base        = flag.String("base", "stable", "The base branch used for PR.")
 	head        = flag.String("head", "master", "The head branch used for PR.")
 	pullRequest = flag.Int("pr", 0, "The Pull request to use.")
+	codecov     = flag.Bool("codecov", false, "Skip code coverage failure")
```

Comment:
Please create a unique flag for those that takes in a comma separated list of context to skip.